server/cli/flags.go

```diff
@@ -78,6 +78,11 @@ func initFlags(ctx *server.Context) {
 
 	flag.Int64Var(&ctx.CacheSize, "cache-size", ctx.CacheSize, "total size in bytes for "+
 		"caches, shared evenly if there are multiple storage devices.")
+
```

Comment:
I think we should nix this as a flag. Let's be allergic to adding new dials exposed to the end user.